app-backend/db/src/test/scala/com/azavea/rf/database/TaskDaoSpec.scala

```diff
@@ -1505,4 +1505,187 @@ class TaskDaoSpec
       }
     }
   }
+
+  test("updating reviews of tasks should update review status of their parent") {
+    check {
+      forAll {
+        (
+            userCreate: User.Create,
+            orgCreate: Organization.Create,
+            platform: Platform,
+            projectCreate: Project.Create,
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate,
+            annotationProjectCreate: AnnotationProject.Create
+        ) =>
+          {
+            val connIO =
+              for {
+                (dbUser, _, _, dbProject) <- insertUserOrgPlatProject(
+                  userCreate,
+                  orgCreate,
+                  platform,
+                  projectCreate
+                )
+                dbAnnotationProj <- AnnotationProjectDao
+                  .insert(
+                    annotationProjectCreate.copy(
+                      projectId = Some(dbProject.id)
+                    ),
+                    dbUser
+                  )
```

Comment:
I think you can get by with just a `User.Create` here, which should make the test faster and help scalacheck out in case something actually goes wrong (it'll have way less stuff to shrink if it doesn't have to guess whether the problem was caused by the org / platform / project)